content/learn/quick-start/getting-started/setup.md

```diff
@@ -246,6 +246,33 @@ channel = "nightly"
 
 For more information, see [The rustup book: Overrides](https://rust-lang.github.io/rustup/overrides.html#the-toolchain-file).
 
+#### Cranelift
+
+This uses a new nightly-only codegen that is about 30% faster at compiling than LLM. 
+
+To install cranelift on Linux or macOS, run the following.
+```
+rustup component add rustc-codegen-cranelift-preview --toolchain nightly
+```
+
+To activate it for your project, add the following to your `.config/cargo.toml`.
+```toml
+[unstable]
+codegen-backend = true
+
+# Replace `my_program` with the name of your binary.
+[profile.dev.package.my_program]
+codegen-backend = "cranelift"
```

Comment:
You can also use

```toml
[profile.dev]
codegen-backend = "cranelift"

[profile.dev.package."*"]
codegen-backend = "llvm"
```

to compile everything with llvm, but compile the main program and all other workspace members with cranelift.

Reply:
I like your version more, thanks! [`5f9b4e6` (#1523)](https://github.com/bevyengine/bevy-website/pull/1523/commits/5f9b4e6c7e3bec8fa631ad71a71abb14a94adbe2)